.devcontainer/Dockerfile

```diff
@@ -1,136 +1,12 @@
-FROM ubuntu:focal
+ARG OS_ARCH=jammy-x86_64
+ARG BRANCH_NAME=mountain-hydrangea
```

Comment:
Just took a look in the registry, it doesn't look like we tag build images with `main` anymore (last time was around January), though maybe we should start again.

Either way, the scripts do set the `.env` automatically for all of the ARGs the dockerfile needs, so maybe it would be best to just remove the defaults entirely?